.github/workflows/ci.yml

```diff
@@ -1,24 +1,46 @@
+name: CI
 on:
   push:
-    branches: [master, release-*]
-    tags: ["*.*.*"]
+    branches: 
+      - master
+      - release-*
   pull_request:
+  workflow_dispatch:
 env:
   DOTNET_NOLOGO: true
 jobs:
-  ci:
-    runs-on: windows-2019
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: windows-2019
+            name: Windows
+          - os: ubuntu-20.04
```

Comment:
We haven't used matrix builds for this project yet but I guess it doesn't hurt given the package itself will also be used on linux builds.